### PR TITLE
Improve generated graphql types

### PR DESCRIPTION
**Pull request by eseliger**
- Make BigInt not be any
- Use the correct import paths for stuff from shared

### CODE REVIEW

File: client/shared/dev/generateGraphQlOperations.js [hunk 1]
```diff
@@ -64,6 +64,7 @@ async function generateGraphQlOperations() {
           GitObjectID: 'string',
           JSONCString: 'string',
           PublishedValue: "boolean | 'draft'",
```
Comment (felixfbecker) [inline client/shared/dev/generateGraphQlOperations.js:66]: drive-by: This is... interesting? Why not make this an enum?

**Review by felixfbecker — APPROVED** [review]

File: client/shared/dev/generateGraphQlOperations.js (hunk 1 above)
Comment (eseliger) [reply client/shared/dev/generateGraphQlOperations.js:66]: Because the original spec before we introduced draft changesets was `boolean` and we don't break compatibility.
